okhttp-client/src/main/scala/org/http4s/client/okhttp/OkHttpBuilder.scala

```diff
@@ -41,21 +41,25 @@ import scala.util.control.NonFatal
   */
 sealed abstract class OkHttpBuilder[F[_]] private (
     val okHttpClient: OkHttpClient,
-    val blockingExecutionContext: ExecutionContext
+    val blocker: Blocker
 )(implicit protected val F: ConcurrentEffect[F], cs: ContextShift[F])
     extends BackendBuilder[F, Client[F]] {
   private[this] val logger = getLogger
 
   private def copy(
       okHttpClient: OkHttpClient = okHttpClient,
-      blockingExecutionContext: ExecutionContext = blockingExecutionContext
-  ) = new OkHttpBuilder[F](okHttpClient, blockingExecutionContext) {}
+      blocker: Blocker = blocker
+  ) = new OkHttpBuilder[F](okHttpClient, blocker) {}
 
   def withOkHttpClient(okHttpClient: OkHttpClient): OkHttpBuilder[F] =
     copy(okHttpClient = okHttpClient)
 
+  def withBlock(blocker: Blocker): OkHttpBuilder[F] =
```

Comment:
Since the deprecation warning below refers to `withBlocker` I guess this method should have been called `withBlocker` too. Note the missing `er`.

Reply:
Argh, that's on me. Good catch.